test/integration/ProjectBrowserTest.kt

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (C) 2022 Vaticle
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+// We need to access private function Studio.MainWindowColumn, this allows us to.
+// Do not use this outside of tests anywhere. It is extremely dangerous to do so.
+@file:Suppress("INVISIBLE_REFERENCE", "INVISIBLE_MEMBER")
+
+package com.vaticle.typedb.studio.test.integration
+
+import java.util.UUID
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithText
+import androidx.compose.ui.test.performClick
+import com.vaticle.typedb.studio.state.StudioState
+import org.junit.Rule
+import org.junit.Test
+
+class ProjectBrowserTest {
+    @get:Rule
+    val composeRule = createComposeRule()
+
+    @Test
+    fun createADirectory() {
+        val uuid = UUID.randomUUID().toString()
```

Comment:
Now that we're using the right tool for the job, let's really make the intent crystal clear.

We've made a UUID, but why? What is this UUID for?

If I remember correctly we use it as part of the directory name we create, and in other tests, as a database name to be created. It uniquely identifies the test being run. So how about naming it `testID`?

If so, this change should be propagated to the other places we generate an ID.

Reply:
This has been done. Additionally, we've moved this into an IntegrationTest class.